client/my-sites/domains/domain-management/list/all-domains.jsx

```diff
@@ -682,26 +682,26 @@ class AllDomains extends Component {
 	}
 
 	renderDomainTableFilterButton() {
-		const { context } = this.props;
+		const { context, translate } = this.props;
 
 		const selectedFilter = context?.query?.filter;
 		const nonWpcomDomains = this.mergeFilteredDomainsWithDomainsDetails();
 
 		const filterOptions = [
 			{
-				label: 'All domains',
+				label: translate( 'All domains' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'All Domains' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/domains-filter-button-l10n)